native/src/constants/contentDirection.ts

```diff
@@ -9,3 +9,15 @@ export const contentDirection = (contentLanguage: string): 'row' | 'row-reverse'
   isContentDirectionReversalRequired(contentLanguage) ? 'row-reverse' : 'row'
 export const contentAlignment = (contentLanguage: string): 'right' | 'left' =>
   isContentDirectionReversalRequired(contentLanguage) ? 'right' : 'left'
+export const isRTLText = (text: string): boolean => {
+  const rtlCount = (text.match(/[\u0591-\u07FF\uFB1D-\uFDFD\uFE70-\uFEFC]/g) || []).length
+  // eslint-disable-next-line no-misleading-character-class
+  const ltrCount = (
+    text.match(
+      // eslint-disable-next-line no-misleading-character-class
```

Comment:
Why is this "misleading"?

Reply:
don't ask me i checked the linting rule.. probably due to unicode chars?
https://eslint.org/docs/latest/rules/no-misleading-character-class

Reply:
Wouldn't it be fine, to just have [a-z] and [A-Z]?

Reply:
well there are `ö`and `ä` and some nordic chars and other stuff. i think its no fault to have it more exact.
Here is the bug for this ESLint Rule. It should not mark thus...
https://github.com/eslint/eslint/issues/15080
Doesn't look like it gonna be fixed soon